lib/widgets/drive.coffee

```diff
@@ -34,6 +38,17 @@ getDrives = ->
 			async.reject drives, drivelist.isSystem, (results) ->
 				return callback(null, results)
 
+cleanupList = (previous,currrent) ->
+	removedDrive =
+		name: "#{currrent.device} (#{currrent.size}) - #{currrent.description}"
+		value: currrent.device
+	return _.reject previous, (drive) ->
+		# The reason we use _.pick and pass a custom object in _.isEqual is due to the fact the initial objects (previous, current)
```

Comment:
By opening this file in my editor, I see that the comment width is very long. We usually try not to make any line longer than 80 characters. Can you split the lines of the message to stay inside that limit?